### ISSUE TITLE
[HL2] [Feature Request] Adding HDR effects all around the game

### ISSUE THREAD

**Issue by VladimirVSC** [ValveSoftware/Source-1-Games#3790]
I'm not the first one asking it, I think, but why Xbox360 version has full HDR effects during the whole game and PC original one has only heavily reduced version of them (i'm not even sure where they are exactly). What's the point in HDR option in menu then?

**Comment by shiro-2k** [comment]
I can think of a nice example of HDR right at the startpoint of the map d1_town_05 (exiting Ravenholm) - where the outside light appears brighter than usual due to the player's visual adjustment.

**Comment by VladimirVSC** [comment]
> I can think of a nice example of HDR right at the startpoint of the map d1_town_05 (exiting Ravenholm) - where the outside light appears brighter than usual due to the player's visual adjustment.

But does HDR presented into maps before Ravenholm exit?

**Comment by shiro-2k** [comment]
> 

Close. HDR maps start at Black Mesa East (d1_eli_01) and include the following;
- "We Don't Go To Ravenholm" - d1_town_05
- Highway 17 - d2_coast_01 - d2_coast_03 - d2_coast_04 - d2_coast_05 - d2_coast_07
- Sandtraps - d2_coast_09 - d2_coast_11 - d2_prison_01
- Nova Prospekt - d2_prison_02 - d2_prison_03 - d2_prison_05
- Entanglement  - d2_prison_06 - d2_prison_07

Every other map in the game uses standard LDR. You can use the command mat_force_bloom 1 in the developer console if you want some brighter lighting, though.

**Comment by VladimirVSC** [comment]
> > 
> 
> Close. HDR maps start at Black Mesa East (d1_eli_01) and include the following;
> 
> * "We Don't Go To Ravenholm" - d1_town_05
> * Highway 17 - d2_coast_01 - d2_coast_03 - d2_coast_04 - d2_coast_05 - d2_coast_07
> * Sandtraps - d2_coast_09 - d2_coast_11 - d2_prison_01
> * Nova Prospekt - d2_prison_02 - d2_prison_03 - d2_prison_05
> * Entanglement  - d2_prison_06 - d2_prison_07
> 
> Every other map in the game uses standard LDR. You can use the command mat_force_bloom 1 in the developer console if you want some brighter lighting, though.

I see. Thanks for the information.

**Comment by VladimirVSC** [comment]
> > 
> 
> Close. HDR maps start at Black Mesa East (d1_eli_01) and include the following;
> 
> * "We Don't Go To Ravenholm" - d1_town_05
> * Highway 17 - d2_coast_01 - d2_coast_03 - d2_coast_04 - d2_coast_05 - d2_coast_07
> * Sandtraps - d2_coast_09 - d2_coast_11 - d2_prison_01
> * Nova Prospekt - d2_prison_02 - d2_prison_03 - d2_prison_05
> * Entanglement  - d2_prison_06 - d2_prison_07
> 
> Every other map in the game uses standard LDR. You can use the command mat_force_bloom 1 in the developer console if you want some brighter lighting, though.

But still, first four chapters stay with LDR, and it would be great to see them with HDR too.

**Comment by BreakinBenny** [comment]
If I'm not mistaken, Half-Life 2: Update has HDR in all maps.

**Comment by VladimirVSC** [comment]
> If I'm not mistaken, Half-Life 2: Update has HDR in all maps.

Yes, but Update has its own bugs, and moreover, it is kind of a fan mod. I personally prefer the original game.

**Comment by UAVXP** [comment]
Half-Life 2: Remastered should fix that issue

**Comment by megakarlach** [comment]
Valve added HDR to all maps since 20th anniversary back in 2 months ago, this probably should be closed now.

Btw the HDR effect in 20th anniversary has been also adjusted aswell.

**Comment by kisak-valve** [comment]
Closing per the last comment.
